packages/katello/rubygem-foreman_scc_manager/rubygem-foreman_scc_manager.spec

```diff
@@ -1,32 +1,30 @@
 # template: foreman_plugin
 %global gem_name foreman_scc_manager
 %global plugin_name scc_manager
-%global foreman_min_version 2.1
+%global foreman_min_version 3.7
 
 Name: rubygem-%{gem_name}
-Version: 2.3.0
-Release: 2%{?foremandist}%{?dist}
```

Comment:
@ekohl I think this is a bug introduced by your last patch, rpmdev-bumpspec kills our foremandist :(

Reply:
Oh bleh. I'll come up with a patch.